README.md

```diff
@@ -1,2 +1,63 @@
-# robotflow
-Robot Flow RPA listener for Robot Framework
+# RobotFlow
+
+Controlling Robot Framework RPA flow with florcharts
+
+## Overview
+
+Robot Flow is a RPA tool for Robot Framework (RF). It uses flow charts to control task execution order. Robot Flow based on concept by Mika Kaukoranta <mika.kaukoranta@gmail.com>
+
+RPA designer can draw high level process in flowchart and automator can implement flowchart steps in Robot Framework.
+
+## Required tools
+
+* Robot Framework 3.2 alpha 1 or later (3.2.dev1)
+* Graphml drawing tool like [yEd Graph Editor](https://www.yworks.com/products/yed)
+
+## Installation
+
+Download listener and place it in a suitable location.
+
+## Usage
+
+1. Create high a level RPA flow using flow chart editor.
+1. Write RPA tasks with Robot Framework
+1. Robot Framework task name must match ones in flow chart. Names are case sensitive
+1. Place both files in same directory and check that their names are same but with different suffix
+1. Run Robot Framework with RobotFlow listener `robot -L RobotFlow.py mytask.robot`
+
+## Demo
+
+### Introduction
+
+This repository contains a demo application and RPA scripts that show how this can be used. Application is in demo folder.
+
+Script reads customerid's from input.xml, checks their status from application and writes output to output.xml.
+
+There are two robot files:
+
+* checkcustomer.robot
+  * This script uses RobotFlow to control process execution
+* checkcustomer-rf.robot
+  * This script is written in traditional Robot Framework style and does not use RobotFlow
+
+### Usage instructions
+
+To use demo application:
+
+1. Install required libraries
+1. Start demoserver: `python demoserver.py`
+1. Server opens to localhost port 8080
+1. Run RPA with this command: `robot --rpa --listener ..\listeners\RobotFlow.py checkuser.robot`
```

Comment:
Windows should be able to tolerate forward slashes while Linux will not tolerate backslashes at all.

```suggestion
1. Run RPA with this command: `robot --rpa --listener ../listeners/RobotFlow.py checkuser.robot`
```